exercism/ocr-numbers/ocr-numbers.js

```diff
@@ -0,0 +1,27 @@
+var convert = function(stringNum) {
+  var lookup = {
+    ' _ \n| |\n|_|\n   ': '0',
+    '   \n  |\n  |\n   ': '1',
+    ' _ \n _|\n|_ \n   ': '2',
+    ' _ \n _|\n _|\n   ': '3',
+    '   \n|_|\n  |\n   ': '4',
+    ' _ \n|_ \n _|\n   ': '5',
+    ' _ \n|_ \n|_|\n   ': '6',
+    ' _ \n  |\n  |\n   ': '7',
+    ' _ \n|_|\n|_|\n   ': '8',
+    ' _ \n|_|\n _|\n   ': '9',
+    '    _ \n  || |\n  ||_|\n      ': '10',
+    '   \n| |\n| |\n   ': '?',
+    '       _     _        _  _ \n  |  || |  || |  |  || || |\n  |  ||_|  ||_|  |  ||_||_|\n                           ': '110101100',
+    '       _     _           _ \n  |  || |  || |     || || |\n  |  | _|  ||_|  |  ||_||_|\n                           ': '11?10?1?0',
+    '    _  _     _  _  _  _  _  _ \n  | _| _||_||_ |_   ||_||_|| |\n  ||_  _|  | _||_|  ||_| _||_|\n                              ': '1234567890',
+    '    _  _ \n  | _| _|\n  ||_  _|\n         \n    _  _ \n|_||_ |_ \n  | _||_|\n         \n _  _  _ \n  ||_||_|\n  ||_| _|\n         ': '123,456,789'
```

Comment:
Thanks, Punit. That makes sense. We considered automating this but felt that this met the spec and prioritized exposing ourselves to some more problems/experiences.